code/game/turfs/closed/minerals.dm

```diff
@@ -126,7 +126,13 @@
 
 /turf/closed/mineral/random/air
 	baseturfs = /turf/open/floor/plating/asteroid
-	var/static/datum/gas_mixture/immutable/planetary
+	var/static/gas_mixture/immutable/planetary/GM
```

Comment:
```suggestion
	var/static/datum/gas_mixture/immutable/planetary/GM
```
/gas_mixture/immutable/planetary is not a valid typepath, it needs to start with /datum, since gas_mixture inherits from datum.